app/src/main/res/menu/bottom_nav_menu.xml

```diff
@@ -3,17 +3,17 @@
 
     <item
         android:id="@+id/navigation_search"
-        android:icon="@drawable/ic_home_black_24dp"
+        android:icon="@drawable/ic_search_24px"
         android:title="@string/title_search" />
 
     <item
         android:id="@+id/navigation_barcode"
-        android:icon="@drawable/ic_dashboard_black_24dp"
+        android:icon="@drawable/ic_camera_alt_24px"
         android:title="@string/title_barcode" />
 
     <item
         android:id="@+id/navigation_setting"
-        android:icon="@drawable/ic_notifications_black_24dp"
+        android:icon="@drawable/ic_settings_applications_24px"
```

Comment:
I think the "settings" icon provided by android studio looks better than the "settings_application" icon